apps/src/templates/levelSummary/ResponseMenuDropdown.jsx

```diff
@@ -43,7 +43,10 @@ const ResponseMenuDropdown = ({response}) => {
               <button
                 className={styles.dropdownOption}
                 type="button"
-                onClick={() => setIsOpen(false)}
+                onClick={() => {
+                  setIsOpen(false);
+                  hideResponse(response);
```

Comment:
I am wondering if we need to pass through the whole `response` here (both in the component and in this function) or just the `user_id` of the response to be hidden.